src/Nethermind/Nethermind.Blockchain.Test/TxPools/PendingTransactionComparerTests.cs

```diff
@@ -65,10 +65,10 @@ public static IEnumerable TestCases
         }
         
         [TestCaseSource(nameof(TestCases))]
-        public bool Equals_test(Transaction t1, Transaction t2) => PendingTransactionComparer.Default.Equals(t1, t2);
+        public bool Equals_test(Transaction t1, Transaction t2) => CompetingTransactionEqualityComparer.Instance.Equals(t1, t2);
 
         [TestCaseSource(nameof(TestCases))]
         public bool HashCode_test(Transaction t1, Transaction t2) =>
-            PendingTransactionComparer.Default.GetHashCode(t1) == PendingTransactionComparer.Default.GetHashCode(t2);
+            CompetingTransactionEqualityComparer.Instance.GetHashCode(t1) == CompetingTransactionEqualityComparer.Instance.GetHashCode(t2);
```

Comment:
This name is magic. What is CompetingTransactionEqualityComparer? No way to tell what it does.